scenes/game/game.gd

```diff
@@ -354,6 +353,16 @@ func begin_remote_game(game_start_message):
 	starting_message = game_start_message.duplicate()
 	observer_mode = 'observer_mode' in game_start_message and game_start_message['observer_mode']
 	replay_button_visible = not observer_mode
+	print(game_start_message)
```

Comment:
great for debugging, but prints end up in the console log on the website, so let's not print anything for real.